src/github/middleware.ts

```diff
@@ -9,12 +9,16 @@ import getJiraUtil from "../jira/util";
 import enhanceOctokit from "../config/enhance-octokit";
 import { Context } from "probot/lib/context";
 import { booleanFlag, BooleanFlags } from "../config/feature-flags";
+import { getLogger } from "../config/logger";
 
 // Returns an async function that reports errors errors to Sentry.
 // This works similar to Sentry.withScope but works in an async context.
 // A new Sentry hub is assigned to context.sentry and can be used later to add context to the error message.
 const withSentry = function (callback) {
 	return async (context) => {
+
+		const logger = getLogger("webhook.middleware");
```

Comment:
Make this global to the file so it doesn't have to keep creating the logger on every call

Reply:
done